waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -80,9 +80,7 @@ genSdkReal spec =
     [ genFileCopy [relfile|api/index.ts|],
       genFileCopy [relfile|api/events.ts|],
       genFileCopy [relfile|core/config.ts|],
-      genFileCopy [relfile|core/auth.ts|],
       genFileCopy [relfile|core/storage.ts|],
-      genFileCopy [relfile|core/stitches.config.ts|],
```

Comment:
This file needs access to `stitches/react` which  we only install for apps that use auth.

In other words, this file failed the build for apps that don't use auth because of a missing dependency.

Reply:
Stiches is a way to style React component in general, but used only for our Auth UI components. It makes sense to put it somewhere related to auth for now, but in the future we might want to move it 👍